storage/statedb/database.go

```diff
@@ -358,9 +358,14 @@ func (db *Database) TrieNodeCache() TrieNodeCache {
 	return db.trieNodeCache
 }
 
-// GetTrieNodeLocalCacheLimit returns the byte size of trie node cache.
-func (db *Database) GetTrieNodeLocalCacheLimit() int {
-	return db.trieNodeLocalCacheLimit
+// GetTrieNodeCacheConfig returns the configuration of TrieNodeCache
+func (db *Database) GetTrieNodeCacheConfig() TrieNodeCacheConfig {
+	return db.trieNodeCacheConfig
+}
+
+// GetTrieNodeLocalCacheByteLimit returns the byte size of trie node cache.
+func (db *Database) GetTrieNodeLocalCacheByteLimit() uint64 {
+	return uint64(db.trieNodeCacheConfig.FastCacheSizeMB) * 1024 * 1024
```

Comment:
Before change, trieNodeLocalCacheLimit is initialized as cacheConfig.FastCacheSizeMB, and it is returned without multiply. But in the change, you multiply it. Could you explain why you did it?

Reply:
Good point. The previous code was wrong. The method had returned "Byte" at first but changed to "MB" with my mistake.

Reply:
Understood 👍